Gemfile

```diff
@@ -6,6 +6,6 @@ if !ENV['TRAVIS']
   gem 'yard',   require: false
 end
 
-gem 'lotus-utils', '~> 0.5', require: false, github: 'lotus/utils', branch: '0.5.x'
+gem 'lotus-utils', '~> 0.5', require: false, github: 'theocodes/utils', branch: 'same-strings-join-fix'
```

Comment:
I didn't change this back on purpose so people don't forget to point it at my fix on utils before running tests etc.